wikis.yml

```diff
@@ -7,3 +7,13 @@ en.wikipedia:
 
     # The category that the individual reports should be placed in.
     category: 'Category:Lists of popular pages by WikiProject'
+
+ar.wikipedia:
+    # The 'index' page, which lists all the configured WikiProjects in a formatted table.
+    index: 'ويكيبيديا:قائمة الصفحات الأكثر مشاهدة حسب مشروع الويكي'
+
+    # The configuration page, where you add configuration for individual WikiProjects.
+    config: 'User:Community Tech bot/Popular pages config.json'
```

Comment:
Note this can be anywhere. Would something like `ويكيبيديا:قائمة الصفحات الأكثر مشاهدة حسب مشروع الويكي.json` make more sense, as that's where the index page lives?

Reply:
In arabic wikipedia, we want to use [this page](https://ar.wikipedia.org/wiki/%D9%88%D9%8A%D9%83%D9%8A%D8%A8%D9%8A%D8%AF%D9%8A%D8%A7:%D9%82%D8%A7%D8%A6%D9%85%D8%A9_%D8%A7%D9%84%D8%B5%D9%81%D8%AD%D8%A7%D8%AA_%D8%A7%D9%84%D8%A3%D9%83%D8%AB%D8%B1_%D9%85%D8%B4%D8%A7%D9%87%D8%AF%D8%A9_%D8%AD%D8%B3%D8%A8_%D9%85%D8%B4%D8%B1%D9%88%D8%B9_%D8%A7%D9%84%D9%88%D9%8A%D9%83%D9%8A) as index page. a result like the task performed by MaraBot.

Reply:
Right, I meant the configuration page. Would you not at least prefer "Popular pages config" to be in Arabic? I suggested `ويكيبيديا:قائمة الصفحات الأكثر مشاهدة حسب مشروع الويكي.json` because that seemed like a natural place to put the configuration page, sharing the same name as the index page. It's up to you! `User:Community Tech bot/Popular pages config.json` is also fine :)